# Use non-system prefix to preserve a typical install directory structure
# Davide Vanzo - Vanderbilt University
diff -ru utf8proc-2.2.0.orig/Makefile utf8proc-2.2.0/Makefile
--- utf8proc-2.2.0.orig/Makefile	2018-10-26 11:55:37.038812532 -0500
+++ utf8proc-2.2.0/Makefile	2018-10-26 11:57:27.126812414 -0500
@@ -33,9 +33,8 @@
 endif
 
 # installation directories (for 'make install')
-prefix=/usr/local
-libdir=$(prefix)/lib
-includedir=$(prefix)/include
+libdir=/lib
+includedir=/include
 
 # meta targets
 
